src/main/java/models/standard/war/WarHand.java

```diff
@@ -0,0 +1,24 @@
+package models.standard.war;
+
+import models.Hand;
+import models.standard.StandardPlayingCard;
+import models.standard.StandardPlayingCard.*;
+
+import java.util.Stack;
+
+public class WarHand extends Hand<StandardPlayingCard, Stack<StandardPlayingCard>> {
+
+  private Stack<StandardPlayingCard> cards = new Stack<>();
```

Comment:
`Hand` has the `cards` property doesn't it? You can cut this property in the children